docs/locale/ko/LC_MESSAGES/header/header.po

```diff
@@ -0,0 +1,210 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2025, Lablup Inc.
+# This file is distributed under the same license as the Backend.AI WebUI
+# User Guide package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2025.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Backend.AI WebUI User Guide\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2025-04-08 16:15+0900\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: SungChul Hong <sungchul@lablup.com>\n"
+"Language: ko\n"
+"Language-Team: ko <LL@li.org>\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.14.0\n"
+
+#: ../../header/header.rst:3
+msgid "Top Bar Features"
+msgstr "상단 바 기능"
+
+#: ../../header/header.rst:7
+msgid "The tob bar includes various features that support use of the WebUI."
+msgstr "상단 바에는 WebUI 사용을 지원하는 다양한 기능이 포함되어 있습니다."
+
+#: ../../header/header.rst:-1
+msgid "WebUI Top Bar"
+msgstr ""
+
+#: ../../header/header.rst:15
+msgid "Project selector"
+msgstr "프로젝트 선택기"
+
+#: ../../header/header.rst:19
+msgid ""
+"Users can switch between projects using the project selector provided in "
+"the top bar. By default, the project that user currently belongs to is "
+"selected. Since each project may have different resource policies, "
+"switching projects may also change the available resource policies."
+msgstr "사용자는 상단 바의 프로젝트 선택기를 통하여 사용자의 현재 프로젝트를 선택할 수 있습니다. "
+"각 프로젝트 별로 다른 자원 정책을 가질 수 있으므로, 프로젝트를 변경할 경우 가용 가능한 자원 정책이 변경될 수 있습니다. "
+
+#: ../../header/header.rst:-1 ../../header/header.rst:24
+msgid "Notification"
+msgstr "이벤트 알림"
+
+#: ../../header/header.rst:28
+msgid ""
+"The bell shape button is the event notification button. Events that need "
+"to be recorded during WebUI operation are displayed here. When background"
+" tasks are running, such as creating a compute session, you can check the"
+" jobs here. When the background task is finished."
+msgstr "종 모양의 버튼은 이벤트 알림 버튼입니다. WebUI 동작 과정에서 기록이 필요한 이벤트 내역이 여기에 표시됩니다. "
+"예를 들어, 연산 세션의 생성을 위한 백그라운드 작업이 진행되고 있는 경우, 이벤트 알림에서 해당 내용을 확인할 수 있습니다. "
+
+#: ../../header/header.rst:-1 ../../header/header.rst:39
+msgid "Theme mode"
+msgstr "테마 모드"
+
+#: ../../header/header.rst:43
+msgid ""
+"You can change the theme mode of the WebUI via the dark mode icon on the "
+"right side of the header."
+msgstr "상단 바 우측에 있는 다크모드 아이콘을 통하여 WebUI의 테마를 변경할 수 있습니다. "
+
+#: ../../header/header.rst:52
+msgid "Help"
+msgstr "도움말"
+
+#: ../../header/header.rst:56
+msgid ""
+"There is a question mark icon at the lower right side of the header. "
+"Click this icon to access the web version of this guide document."
+msgstr "상단 바 우측의 물음표 버튼을 통하여, 본 가이드 문서의 웹 버전에 접속할 수 있습니다. "
```

Comment:
It would be great to include content that matches the help information for the current page.